### ISSUE TITLE
Line 20, Commands.py

### ISSUE THREAD

**Issue by TheOneWhoMade** [LunarWatcher/NN-chatbot#7]
`sites: list() = []`

Throws a syntax error:
```
 sites: list() = []
         ^
SyntaxError: invalid syntax
```

**Comment by LunarWatcher** [comment]
Which version of python?

**Comment by TheOneWhoMade** [comment]
3.4.2

**Comment by TheOneWhoMade** [comment]
Thats the only error

**Comment by TheOneWhoMade** [comment]
I am truly sorry for having to make you force it to be backwards compatable.

**Comment by LunarWatcher** [comment]
Works in Python 3.6, if 3.5 was the offender this would be considered fixed. 3.4 and lower isn't supported atm

**Comment by TheOneWhoMade** [comment]
Oh screw it.
`sudo apt-get install python3.6`

**Comment by LunarWatcher** [comment]
And [this](https://docs.python.org/3/library/typing.html) is why. Type hints aren't added until 3.5

**Comment by TheOneWhoMade** [comment]
Yes, i get that. But what i'm confused about is the fact that that linux command up there failed. It said there was no package.

**Comment by TheOneWhoMade** [comment]
Is there a specific version of 3.6 you are using?

**Comment by LunarWatcher** [comment]
I'm using 3.6.4, but python uses 3 digits. 3.6 is 3.6.0

**Comment by TheOneWhoMade** [comment]
So that might do it.
`sudo apt-get install python3.6.0`
?

**Comment by TheOneWhoMade** [comment]
Still no package found.

**Comment by LunarWatcher** [comment]
Dunno. I just used the installer

**Comment by TheOneWhoMade** [comment]
Remember, I am using linux

**Comment by LunarWatcher** [comment]
Then you have to find the solution that works for your OS.

**Comment by TheOneWhoMade** [comment]
Yeh. I also have to find a sollution that works for the restrictions on my user.

**Comment by TheOneWhoMade** [comment]
https://etechroom.com/how-to-install-python3-on-linux-using-command-line-interface/ 
working so far, unlike the others

**Comment by TheOneWhoMade** [comment]
BTW we should clean this up. (Please dont remove my comments I can remove them)
